rustfmt.toml

```diff
@@ -0,0 +1,20 @@
+# # = are nightly features
+max_width = 80
+reorder_imports = true
+#combine_control_expr = false
+#format_code_in_doc_comments = true
+#format_macro_matchers = true
+hard_tabs = false
+#merge_imports = true
+#normalize_comments = true
+#wrap_comments = true
+#normalize_doc_attributes = true
+#overflow_delimited_expr = true
+#reorder_impl_items = true
+#struct_lit_single_line = true
+use_field_init_shorthand = true
+use_try_shorthand = true
+#where_single_line = true
+#imports_layout = "Vertical"
+fn_args_layout = "Vertical"
+#control_brace_style = "ClosingNextLine"
```

Comment:
I would keep rustfmt defaults, I don't see the point of overriding them. It also opens the door to bikeshedding about the code style (like what I'm doing here).

Reply:
Hehe, ok, so you think it's better to remove the whole rustfmt.toml then or comment out the rest?

Reply:
Remove everything. let's reduce boilerplate!

Reply:
/done